src/gui/basegraphgl.h

```diff
@@ -80,6 +83,9 @@ class BaseGraphGL : public QOpenGLWidget, public GraphGLInterface
     void paint(QPaintDevice* device, bool paintBackground) const;
 
     inline int currStep() const { return m_currStep; }
+    inline void setInspector(FullInspector* inspector);
+    std::map<int, Node> m_selectedNodes; //TODO: This shouldn't be public
+//    void updateFullInspector();
```

Comment:
I see the TODO note, but we should never ever push code with public members like that...
In any case, I see that `m_selectedNodes` is not being used anyway, so I think it could be removed